shepherd/Goal.py

```diff
@@ -47,13 +48,23 @@ def score(self, alliance):
             scorring Alliance is the owning alliance.
                 alliance - Alliance object representing the scoring team
         """
-        pass
+        if self.owner is alliance:
+            alliance.change_score(self.value * self.calc_multiplier() *\
+                alliance.alliance_multiplier)
 
     def calc_multiplier(self):
         """ returns an integer that represents the overall multiplier for this
             goal, by checking each timer in the goal
         """
-        pass
+        if (self is None or self.gold_two_x_timer is None or
+                self.blue_two_x_timer is None or self.gold_zero_x_timer is None or
+                self.blue_zero_x_timer is None):
+            return 1
+
+        return (self.gold_two_x_timer.is_running() and 2 or 1) *\
```

Comment:
Even though this works fine, I think there should be a more clear way to incorporate this behavior instead of relying on the reader to do boolean logic in their head.

Reply:
-1 to boolean logic